web_external/js/views/widgets/WMSFeatureInfoWidget.js

```diff
@@ -0,0 +1,110 @@
+minerva.views.WmsFeatureInfoWidget = minerva.View.extend({
+
+    callInfo: function (layer_idx, coords) {
+        if (this.layers.length > 0) {
+            var url = this.getUrl(layer_idx, coords);
+            var layer_name = this.layers[layer_idx].layerName;
+            var panel = this;
+            $.ajax({
+                url: url,
+                jsonp: false,
+                async: false,
+                jsonpCallback: 'getLayerFeatures',
+                error: function (a, b, c) {
+                    console.log(c);
+                }
+            }).done(function (data) {
+                //panel.content += data;
+                if ('features' in data && data.features.length > 0) {
+                    var layer_div = document.createElement('div');
```

Comment:
I would prefer it we could create all of the HTML in jade, perhaps some restructuring would be necessary to make the render happen upon the async return.  I'd rather you not change this now, because we are probably going to want to sit on this for a bit, think about it, and then decide on where to take it--we can rewrite it then.  If you have some specifics about why this couldn't be done in the jade, let me know.  I'll add an issue to revise this at a later date.